spaces/reb2/index.json

```
@@ -0,0 +1,11 @@
+{
+  "key": "reb2",
+  "symbol": "REB2"
```

You miss a comma ',' at the end of symbol line. 
```suggestion
  "symbol": "REB2",
```